src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -99,6 +99,28 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
     // (1 commit of `allow-ownership-link` and 3 commits of `call-ws` are all gone)
     Assert.assertEquals(13, branchAndCommitRowsCount)
 
+    project.checkoutBranch("allow-ownership-link")
```

Comment:
Wow there's quite a lot of checking out here in this test... actually, since UI test execution timeout is a bottleneck in CI, I'd just leave a **single invocation** to each of `checkout{Next,Prev,Parent}Branch`